draft-ietf-bidding-and-auction-services.md

```diff
@@ -978,15 +992,48 @@ response from Bidding and Auction Services. It takes as input the
      parsed as a [URL], returning failure if there is an error.
 1. If `response["adMetadata"]` exists and is a string set
    `processed response["ad metadata"]` to `response["adMetadata"]`.
-1. If `response["buyerReportingId"]` exists and is a string, set

```

Comment:
We probably should keep the comma and selectedBuyerAndSellerReportingId parts.

Reply:
added back. Weird